.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ defaults:
     shell: "bash"
 
 env:
-  CI_PRERELEASE: "${{ github.event_name == 'push' }}"
+  CI_PRERELEASE: "${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}"
```

Comment:
This shouldn't be necessary; 'push' events only trigger on the master branch, as seen a few lines above.

Reply:
They do in normal conditions, but when implementing things in CI one might want to run the workflows on push (before putting a PR up), as in e.g. [this commit](28534ee9036cab7eb3048ca7e7fe698d198337f7) - so this an additional safeguard to make sure one does not accidentally pushes a prerelease.